wal/wal.go

```diff
@@ -330,27 +330,33 @@ func (w *WAL) sync() error {
 	return err
 }
 
-// ReleaseLockTo releases the locks w is holding, which
-// have index smaller or equal to the given index.
+// ReleaseLockTo releases the locks w is holding so that
+// w only holds one lock that has an index smaller than
+// the given index.
```

Comment:
don't follow this sentence easily. I thought it doesn't hold the lock with bigger index at the first read.